mlflow/mleap.py

```diff
@@ -21,13 +25,14 @@
 from mlflow.models.utils import _save_example
 from mlflow.utils import reraise
 from mlflow.utils.file_utils import path_to_local_file_uri
-from mlflow.utils.annotations import keyword_only
+from mlflow.utils.annotations import keyword_only, deprecated
 
 FLAVOR_NAME = "mleap"
 
 _logger = logging.getLogger(__name__)
 
 
+@deprecated(alternative="mlflow.onnx", since="2.6.0")
```

Comment:
Is `mlflow.onnx` alternative?

Reply:
Same question, I think onnx cannot directly support exporting spark model.

Reply:
Any change to this ? How does one export Spark Xgboost model to ONNX ?